Tools/TestWebKitAPI/Tests/WTF/Bitmap.cpp

```diff
@@ -271,13 +271,13 @@ void testBitmapConcurrentTestAndClear()
     for (size_t i = 0; i < size; ++i)
         ASSERT_EQ(bitmap1.get(i), expectedBits1[i]);
     for (size_t i = 0; i < size; ++i)
-        ASSERT_EQ(bitmap1.testAndClear(i), expectedBits1[i]);
+        ASSERT_EQ(bitmap1.concurrentTestAndClear(i), expectedBits1[i]);
```

Comment:
The same bug also exists in `testBitmapConcurrentTestAndSet`.  Please fix that also.